crates/nu-command/src/generators/unfold.rs

```diff
@@ -0,0 +1,188 @@
+use itertools::unfold;
+
+use nu_engine::{eval_block_with_early_return, CallExt};
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Closure, Command, EngineState, Stack};
+use nu_protocol::{
+    Category, Example, IntoInterruptiblePipelineData, IntoPipelineData, PipelineData, ShellError,
+    Signature, Span, Spanned, SyntaxShape, Type, Value,
+};
+
+#[derive(Clone)]
+pub struct Unfold;
+
+impl Command for Unfold {
+    fn name(&self) -> &str {
+        "unfold"
```

Comment:
would it make sense to have this under the `seq` command, as `seq unfold`? 

https://www.nushell.sh/commands/docs/seq.html#frontmatter-title-for-generators

Reply:
While I think `unfold` definitely belongs in the `generators` family, I don't think it belongs under `seq`. Seq's description is to "Output sequences of numbers.". `unfold` is a more general purpose tool. While many examples have focused around sequences of numbers, real-world use will likely be see a lot of paginating APIs and interfacing with external systems.

Reply:
That sounds like a documentation issue with `seq` being "for numbers" only - sequences are a general purpose concept. In my view, for nushell, unfold will always produce a sequence/list/table of zero or more items/rows.

So, since I consider sequences generalized it makes sense to group sequence generators under `seq`. Just my two cents. would love to hear others thoughts on this though.

Reply:
I'm open to hearing others input as well.

IMHO, I think of `seq` as a [math sequence](https://en.wikipedia.org/wiki/Sequence). Given unfold is a more general stream source/builder, I feel that it best belongs in the `generators` family of commands.